app/pods/agendas/template.hbs

```diff
@@ -19,12 +40,176 @@
+                @label={{t "status"}}
+              />
+              <Utils::ThSortable
+                class="auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="created-for.kind.label"
+                @onChange={{this.sortTable}}
+                @label={{t "kind"}}
+              />
+              <Utils::ThSortable
+                class="auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="modified"
+                @onChange={{this.sortTable}}
+                @label={{t "latest-modified"}}
+              />
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {{#if this.isLoadingModel}}
+              <tr>
+                <td
+                  data-test-route-agendas-overview-loader
+                  colspan="5"
+                >
+                  {{t "loading"}}
+                </td>
+              </tr>
+            {{else}}
+              {{#each @model as |agenda|}}
+                <tr
+                  class="auk-u-text-align--center auk-table--interaction-cursor auk-table--clickable-rows"
+                  {{! template-lint-disable no-invalid-interactive }}
+                  {{on "click" (fn this.onClickRow agenda)}}
+                >
+                  <td data-test-route-agendas-overview-row-title>
+                    <span class="auk-o-flex auk-o-flex--vertical-center auk-o-flex--spaced">
+                      {{#if agenda.status.isDesignAgenda}}
+                        <strong>{{t "design-agenda"}}</strong>
+                        <Auk::Pill @skin="active-future">{{agenda.name}}</Auk::Pill>
+                      {{else}}
+                        <strong>{{t "agenda"}}</strong>
+                        <Auk::Pill>{{agenda.name}}</Auk::Pill>
+                      {{/if}}
+                      <span>–</span>
+                      <span>
+                        {{moment-format agenda.createdFor.plannedStart "DD.MM.YYYY"}}
+                      </span>
+                    </span>
+                  </td>
+                  <td>
+                    {{#if agenda.createdFor.isFinal}}
+                      <Auk::Pill>{{t "closed"}}</Auk::Pill>
+                    {{else}}
+                      <Auk::Pill @skin="active">{{t "opened"}}</Auk::Pill>
+                    {{/if}}
+                  </td>
+                  <td data-test-route-agendas-overview-row-kind>
+                    {{agenda.createdFor.kind.label}}
+                  </td>
+                  <td>
+                    {{#if (is-pending agenda)}}
+                      <div
+                        class="auk-skeleton auk-skeleton--rectangle"
+                        role="alert"
+                        aria-busy="true"
+                      ></div>
+                    {{else}}
+                      {{moment-format
+                        agenda.modified
+                        "DD.MM.YYYY — HH:mm"
```

Comment:
Date format has been decided to be `DD-MM-YYYY ...` (usertest is not yet in line with the default that has been decided recently)